README.md

```diff
@@ -111,7 +118,7 @@ Set environment variables for each environment in the [app.py](./app.py) file:
 environment_variables = {
     "VPC_CIDR": "10.254.192.0/24",
     "FQDN": "dev.app.io",
-    "CERTIFICATE_ARN": "arn:aws:acm:us-east-1:XXXXXXXXXXX:certificate/0e9682f6-3ffa-46fb-9671-b6349f5164d6",
+    "CERTIFICATE_ID": "0e9682f6-3ffa-46fb-9671-b6349f5164d6",
```

Comment:
now that we are co-locating agora and modelexplorer in one aws account we need to be more specific with the below `CostCenter` tag.  please update both to each specific project's cost center id.

Reply:
new cert ids are in issue IT-4426

Reply:
ohh right, that's only in the readme.  nah, we should leave it as generic as possible.